src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -1310,12 +1305,12 @@ static void perform_stream_op_locked(void* stream_op,
     }
   }
 
-  if (s->context != nullptr) {
-    MaybeRecordTransportAnnotation(
-        s, absl::StrFormat(
-               "perform_stream_op_locked[s=%p; op=%p]: %s; on_complete = %p", s,
-               op, grpc_transport_stream_op_batch_string(op, true).c_str(),
-               op->on_complete));
+  if (auto* tracer = CallTracerIfEnabled(s)) {
+    tracer->RecordAnnotation(absl::StrCat(
+        "perform_stream_op_locked[s=0x", absl::Hex(s, absl::kZeroPad16),
```

Comment:
Same suggestion here regarding `%p` for consistency.

Reply:
Done